test/react-checkbox-group_spec.js

```diff
@@ -25,11 +25,13 @@ describe('ReactCheckboxGroup', function() {
     }).to.throw();
   });
 
-  it('Renders correctly when `Checkbox` is not a direct child of `CheckboxGroup` and `checkboxDepth` is passed', function() {
+  it('Renders correctly when `Checkbox` components on a different nesting levels and one of the child is null', function() {
```

Comment:
Thanks also for updating the tests! Can you please change this to be a separate test case for the `null`?